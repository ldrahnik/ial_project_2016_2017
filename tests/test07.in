A 4 B
B 8 C
C 7 D
D 9 E
E 10 F
F 2 G
G 1 H
H 8 A
B 11 G
C 2 I
I 6 G
I 7 H 
C 4 F
D 14 F
