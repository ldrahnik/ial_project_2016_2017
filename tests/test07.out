Dijkstra: A -(4)> B -(11)> G
